### PR TITLE
[18.06] prep for 18.06.0-ce-rc1

### PR DESCRIPTION
**Pull request by jose-bigio**
changelog and components.conf update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,163 @@
...
+
+### Important notes about this release
+
+- TODO: Notice here for release cycle change
+
+### Builder
+
+* Builder: fix layer leak on multi-stage wildcard copy. [moby/moby#37178](https://github.com/moby/moby/pull/37178)
+* Fix parsing of invalid environment variable substitution . [moby/moby#37134](https://github.com/moby/moby/pull/37134)
+* Dockerbuilder: use the arch info from base image. [moby/moby#36816](https://github.com/moby/moby/pull/36816)
```
Comment (tonistiigi) [inline CHANGELOG.md]: s/Dockerbuilder/Builder/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1 +1,163 @@
...
+* Don't fail to start daemon if builder source is not available. [moby/moby#37280](https://github.com/moby/moby/pull/37280)
```
Comment (tonistiigi) [inline CHANGELOG.md]: This is not an change between releases

File: CHANGELOG.md [hunk 3]
```diff
@@ -1 +1,163 @@
...
+* Experimental BuildKit support. [moby/moby#37151](https://github.com/moby/moby/pull/37151)
+* Export BuildArgs type from dockerfile package. [moby/moby#37016](https://github.com/moby/moby/pull/37016)
```
Comment (tonistiigi) [inline CHANGELOG.md]: There is no feature change part of this so probably not needed in changelog

File: CHANGELOG.md [hunk 4]
```diff
@@ -1 +1,163 @@
...
+* Support for experimental BuildKit. [docker/cli#1111](https://github.com/docker/cli/pull/1111)
```
Comment (tonistiigi) [inline CHANGELOG.md]: Same as line 18

File: CHANGELOG.md [hunk 5]
```diff
@@ -1 +1,163 @@
...
+* Vendor: dockerfile parser from buildkit. [moby/moby#37197](https://github.com/moby/moby/pull/37197)
```
Comment (tonistiigi) [inline CHANGELOG.md]: Same as line 19

File: CHANGELOG.md [hunk 6]
```diff
@@ -1 +1,163 @@
...
+- Fix build on OpenBSD by defining Self(). [moby/moby#37301](https://github.com/moby/moby/pull/37301)
```
Comment (tonistiigi) [inline CHANGELOG.md]: Not a builder feature (about building dockerd)

File: CHANGELOG.md [hunk 7]
```diff
@@ -1 +1,163 @@
...
+- Fix the target name issue for multi-stage build. [moby/moby#36960](https://github.com/moby/moby/pull/36960)
```
Comment (tonistiigi) [inline CHANGELOG.md]: "Fix handling uppercase targets names in multi-stage builds"

Comment (tonistiigi) [inline CHANGELOG.md]: "Fix handling uppercase targets names in multi-stage builds"

File: CHANGELOG.md [hunk 8]
```diff
@@ -1 +1,160 @@
...
+
+### Important notes about this release
+
+- TODO: Notice here for release cycle change
+
+### Builder
+
+* Builder: fix layer leak on multi-stage wildcard copy. [moby/moby#37178](https://github.com/moby/moby/pull/37178)
+* Fix parsing of invalid environment variable substitution . [moby/moby#37134](https://github.com/moby/moby/pull/37134)
+* Builder: use the arch info from base image. [moby/moby#36816](https://github.com/moby/moby/pull/36816) [docker/cli#1111](https://github.com/docker/cli/pull/1111) [moby/moby#37197](https://github.com/moby/moby/pull/37197)
```
Comment (tonistiigi) [inline CHANGELOG.md]: Something wrong in here. Cli links is for "experimental buildkit" line. 37197 is duplicated.

**Comment by andrewhsu** [comment]
@tonistiigi gah, changelog builder items updated in 5f4e312

**Comment by andrewhsu** [comment]
Rolling with what we got for the RC1. Will update changelog again before GA.

**Pull request merged**